docs/user-guide/annotations.md

```diff
@@ -79,3 +79,24 @@ resource "aws_instance" "foo" { # tflint-ignore-file: aws_instance_invalid_type
   instance_type = "t1.2xlarge"
 }
 ```
+
+The `tflint-ignore-file` annotation is also supported in Terraform JSON by
+using a top-level [comment property](https://developer.hashicorp.com/terraform/language/syntax/json#comment-properties):
+
+```json
+{
+  "//": {
+    "tflint-ignore-file": "aws_instance_invalid_type"
+  },
+  "resource": {
+    "aws_instance": {
+      "foo": {
+        "instance_type": "t2.micro"
+      }
+    }
+  }
+}
+```
+
+Note that the `tflint-ignore` annotation is not currently supported in
+Terraform JSON config.
```

Comment:
```suggestion
The `tflint-ignore` annotation is not supported JSON configuration.
```

Avoid "currently" in documentation and similar caveats. Just say how it works right now.